assets/js/blocks/active-filters/block.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useQueryStateByKey } from '@woocommerce/base-hooks';
+import { useMemo, Fragment } from '@wordpress/element';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getAttributeFromTaxonomy } from '../../utils/attributes';
+import { formatPriceRange, renderItem } from './utils';
+import ActiveAttributeFilters from './active-attribute-filters';
+
+/**
+ * Component displaying active filters.
+ */
+const ActiveFiltersBlock = ( {
+	attributes: blockAttributes,
+	isPreview = false,
+} ) => {
+	const [ productAttributes, setProductAttributes ] = useQueryStateByKey(
+		'product-grid',
+		'attributes',
+		[]
+	);
+	const [ minPrice, setMinPrice ] = useQueryStateByKey(
+		'product-grid',
+		'min_price'
+	);
+	const [ maxPrice, setMaxPrice ] = useQueryStateByKey(
+		'product-grid',
+		'max_price'
+	);
+
+	const activePriceFilters = useMemo( () => {
+		if ( ! Number.isFinite( minPrice ) && ! Number.isFinite( maxPrice ) ) {
+			return;
+		}
+		return renderItem(
+			__( 'Price:', 'woo-gutenberg-products-block' ),
+			formatPriceRange( minPrice, maxPrice ),
+			() => {
+				setMinPrice( null );
+				setMaxPrice( null );
+			}
+		);
+	}, [ minPrice, maxPrice, formatPriceRange ] );
+
+	const activeAttributeFilters = useMemo( () => {
+		return (
+			<Fragment>
+				{ productAttributes.map( ( attribute ) => {
+					const attributeObject = getAttributeFromTaxonomy(
+						attribute.attribute
+					);
+					return (
+						<ActiveAttributeFilters
+							attributeObject={ attributeObject }
+							slugs={ attribute.slug }
+							key={ attribute.attribute }
+						/>
+					);
+				} ) }
+			</Fragment>
+		);
+	}, [ productAttributes ] );
+
+	const hasFilters = useMemo( () => {
+		return (
+			productAttributes.length > 0 ||
+			Number.isFinite( minPrice ) ||
+			Number.isFinite( maxPrice )
+		);
+	}, [ productAttributes, minPrice, maxPrice ] );
```

Comment:
This is a pretty cheap calculation, we can likely avoid the overhead of memoization here and drop the use of `useMemo`